Source Main 5.2/source/CComGem.cpp

```diff
@@ -34,7 +34,7 @@ namespace COMGEM
 
 void COMGEM::SendReqUnMix()
 {
-    iUnMixIndex += 12;
+    iUnMixIndex += MAX_EQUIPMENT;
```

Comment:
Not sure if this is better, or if adding the entry to the list of candidates (packed jewels) should already have this offset added.